ando/tools/generator/tests/utils.py

```diff
@@ -1,8 +1,10 @@
 import os
 import tempfile
 import shutil
+import pathlib
+from os import path
```

Comment:
are you sure you need `os.path` in this submodule?

Reply:
updated